deployment/ccip/changeset/cs_jobspec.go

```diff
@@ -27,10 +28,13 @@ func CCIPCapabilityJobspecChangeset(env deployment.Environment, _ any) (deployme
 		return deployment.ChangesetOutput{}, err
 	}
 	// find existing jobs
-	existingSpecs := make(map[string][]string)
+	existingSpecs, err := acceptedJobSpecs(env, nodes)
```

Comment:
I might be missing something, but what is the difference between the logic in `acceptedJobSpecs()` and the implementation between lines 35-55? They seem quite similar.

Reply:
thanks for this! I missed to delete the section after moving it to a func. removed